src/scala/META-INF/native-image/org/pantsbuild/zinc/compiler/reflect-config.json

```diff
@@ -0,0 +1,2414 @@
+[
```

Comment:
Can you add a README or a comment in the BUILD file explaining what this file is, how it was generated, and when it should be re-generated?

Reply:
Done in the README.md in the same directory!